app/src/main/java/org/simple/clinic/ReleaseClinicApp.kt

```diff
@@ -26,6 +27,8 @@ class ReleaseClinicApp : ClinicApp() {
     // for `ControllerStateBase` where we log those.
     val logger = LoggerFactory.getLogger("com.spotify.mobius.ControllerStateBase") as Logger
     logger.level = Level.OFF
+
+    MobiusHooks.setErrorHandler { /* No-op */ }
```

Comment:
What happens to the app in this scenario? Does it continue to run? This might be a bit problematic if the app just stops responding events.

Reply:
If an error happens in the effect handler app will crash, but if it happens in the Update function, it's silently failing so the app will not respond to those events. 

Is there a way we can crash the app without sending the results back to Sentry in this scenario? 🤔

Reply:
Ok, that seems a bit problematic to me. For now, can we just override the `toString()` method of the event that contained PHI to stop it from getting serialized in production?

We can think of a better long term solution later, maybe something like https://mustafaali.net/2018/01/14/Kotlin-data-classes-and-sensitive-information/.

Reply:
Done, for now, I have gone through Sentry issues and redacted the logs that had sensitive info. (only for these errors). Overall this issue seems to be related to some lifecycle-related issues. Will debug further.